scripts/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```

Comment:
default target looks like es3 if not specified.  But, shouldn't matter for examples.

Reply:
Yeah, it's actually for scripts and the scripts are only ever run with `ts-node` which compiles and executes for us so we shouldn't need to dictate a newer target.